app/uk/gov/hmrc/organisationsmatchingapi/platform/controllers/DocumentationController.scala

```diff
@@ -1,47 +0,0 @@
-/*
```

Comment:
This file is needed for the publish. Please can we add it back in?

Reply:
I removed the redundant DocumentationController that had been left in from the original repo - there’s still one in there, the one referred to in the routes. Can we recheck?